SteamTrade/TradeOffer/TradeOffer.cs

```diff
@@ -138,16 +124,16 @@ public bool Send(out string offerId, string message = "")
         /// <param name="token">The token of the partner</param>
         /// <param name="message">Optional message to included with the trade offer</param>
         /// <returns></returns>
-        public bool SendWithToken(out string offerId, string token, string message = "")
+        public int SendWithToken(out string offerId, string token, string message = "", bool debug = false)
         {
             offerId = String.Empty;
             if (TradeOfferId == null)
             {
-                return Session.SendTradeOfferWithToken(message, PartnerSteamId, this.Items, token, out offerId);
+                return Session.SendTradeOfferWithToken(message, PartnerSteamId, this.Items, token, out offerId, debug);
```

Comment:
Does this method even exist?

https://github.com/Jessecar96/SteamBot/blob/master/SteamTrade/TradeOffer/OfferSession.cs#L221